lib/rules/no-dispatch-and-selectors-imports-from-redux.js

```diff
@@ -0,0 +1,51 @@
+/**
+ * @fileoverview useSelector and useDispatch should not be imported in webApp
+ * @author Marcin Brumer
+ */
+'use strict';
+//------------------------------------------------------------------------------
+// Rule Definition
+//------------------------------------------------------------------------------
+module.exports = {
+    meta: {
+        docs: {
+            description: "useSelector and useDispatch should not be imported in webApp",
+            category: 'Stylistic Issues',
+            recommended: false
+        },
+        fixable: null, // or "code" or "whitespace"
+        schema: []
+    },
+
+    create: function(context) {
+        return {
+            ImportDeclaration(node) {
+                const fileName = context.getFilename();
+                if (fileName.includes('.test.') || !fileName.includes('/webApp/')) {
+                    return undefined;
+                }
+                const correctVariables = [];
+                const incorrectVariables = [];
+                context.getDeclaredVariables(node).forEach(item => {
```

Comment:
would be faster to use for loop and break on first incorrect occurrence

Reply:
Sure, will fix that

Reply:
But wait, if I break on first incorrect occurrence, then I will not know if there is second one?

Reply:
yeah, hard to say what's preferable as file would be flagged. But we can stay with your solution